test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/british-or-irish.txt

```diff
@@ -0,0 +1,13 @@
+Is the person:
```

Comment:
I don't understand very well this commit. 
You are adding an option that does not exist in the flow, so the artefacts do not make much sense to me:

The copy of this artefact `test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/british-or-irish.txt` only have one option `eea`.

Preparatory commits make sense for options that already exist in the flow and are going to be updated, not for options that are going to be created in the future.